tests/test_ping_json.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+
+def json_health_check():
+    public_health = "http://localhost:3000/health"  # Access public app health page
+    admin_health = "http://localhost:3001/health"  # Access admin app health pages
+
+    try:
+        public_response = requests.get(public_health)
+        admin_response = requests.get(admin_health)
+
+        if public_response.status_code == 200:
```

Comment:
instead of using if else and print statements, although its good to log in tests, you can just use the 'assert' keyword